helpers/packet_storm_crawler.sh

```diff
@@ -11,14 +11,17 @@
 #
 # EMBA is licensed under GPLv3
 #
-# Author(s): Michael Messner
+# Author(s): Michael Messner, Endri Hoxha
 
 # Description:  Update script for packetstorm PoC/Exploit collection
 
 URL="https://packetstormsecurity.com/files/tags/exploit/page"
 LINKS="packet_storm_links.txt"
 SAVE_PATH="/tmp/packet_storm"
 EMBA_CONFIG_PATH="./config/"
+TAGS_CVES=""
+NO_DUP_LINKS=""
+NUMBER_OF_PAGES=$(lynx -dump "${URL}" | grep -E "Page 1 of " | sed -r 's/.*of ([0-9])\,/\1/')
```

Comment:
should we do a check to ensure we have a number of pages here?
if ! [[ "${NUMBER_OF_PAGES}" -gt 0 ]]; then
  echo "[-] Number of pages could not be extracted ... exit now"
  exit 1
fi

Reply:
We could, but the number of pages is always present

Reply:
Please include such a quick check.

Reply:
done